neeruram1/Crate
# Backend testing

### Type of change made:
- [x] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features

### Description
* We completed testing the queries and mutations for a User. We also completed testing for the SurveyImage query. We seeded the database with SurveyImages, and updated and corrected the seed files. 


### How should this be manually tested?
* This branch completed testing, so manual testing is not required. However, to manually test the seed files run the api server and verify the seed data in the GraphiQL web interface. 

### Any background context you want to provide?
* cd into the api directory and run the command `npm run setup`
If you are still having trouble seeding, drop the database and recreate it and reseed. 

### What are the relevant tickets?
* Resolves #36 
Resolves #19 
Resolves #35 
Resolves #55 

### Checklist

### Questions:
